sufia.gemspec

```diff
@@ -49,6 +49,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'flipflop', '~> 2.2'
   spec.add_dependency 'jquery-datatables-rails', '~> 3.4.0'
   spec.add_dependency 'rdf-rdfxml'
+  spec.add_dependency 'sprockets-rails', '~> 3.1.1'
```

Comment:
It appears that 3.2 was released in Sept 2016. So it's likely no-one encountered this on this branch. https://rubygems.org/gems/sprockets-rails/versions/3.2.0